pyproject.toml

```diff
@@ -256,6 +260,6 @@ networktest = "pytest --cov --cov-report=xml --cov-report=term --durations=0"
 coincident = { path = ".", editable = true }
 
 [tool.pixi.feature.docs.tasks]
-docs = "python -m sphinx -W --keep-going -T -b html -d docs/_build/doctrees -D language=en docs docs/_build/html"
+docs = "python -m sphinx --keep-going -T -b html -d docs/_build/doctrees -D language=en docs docs/_build/html"
```

Comment:
removed `-W` (--warning-is-error) to avoid non-zero exit codes and therefore always generate a preview. The downside is that it'll be easier to miss problems since the docs will always "successfully build". So a manual check of docs and logs is going to be necessary!